revolv/static/css/screen.css

```diff
@@ -10750,6 +11244,86 @@ header.edit-project-header > .container > h1 {
   .header.after-scroll-header .rights {
     padding: 33px 8px 0 0;
   }
+
+/* 30-Jun */
```

Comment:
Same thing with this comment. Please update it to describe what the styles below apply to.